targets/TARGET_Cypress/TARGET_PSOC6/cy_us_ticker_api.c

```diff
@@ -54,6 +54,23 @@ static void cy_us_ticker_irq_handler(MBED_UNUSED void *arg, MBED_UNUSED cyhal_ti
 void us_ticker_init(void)
 {
     if (!cy_us_ticker_initialized) {
+
+#ifdef TARGET_TFM
```

Comment:
We have to prevent Mbed OS from using the timer the regression test application would use. Any suggestions on how to do this from an Mbed OS application, rather than Mbed OS itself, would be much appreciated! Thanks.

Reply:
Do you have the application code that I can take a look at? I expect you don't want to put any Cypress specific code there, but I don't see any of this help without doing so. The reserved times are just at the Cypress layer. Nothing is actually configuring them here.